VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -612,8 +612,16 @@ Function Set-EsxiCertificateMode {
         $certModeSetting = Get-AdvancedSetting "vpxd.certmgmt.mode" -Entity $vCenterServer.connection
         if ($certModeSetting.value -ne $mode) {
             Set-AdvancedSetting $certModeSetting -Value $mode -confirm:$false
-            Write-Output "Certificate Management Mode is set to $mode on the vCenter Server instance $($vCenterServer.details.fqdn)."
-            Write-Output "Please restart the vCenter Server services for the change to take effect. See the vCenter Server Configuration documentation for information about restarting services."
+            # Restart "VMware Certificate Authority Service" and "VMware Certificate Management Service"
+            Invoke-RestartService -Service "certificateauthority"
```

Comment:
```suggestion
            Write-Output "Restarting vCenter Server services for the change to take effect."
            Invoke-RestartService -Service "certificateauthority"
```

Reply:
addressed in commit 9d2c103e0541e6b077e54e83e2aa0db4e5f07a40